src/store/constants.js

```diff
@@ -23,4 +23,3 @@ export const UNIFIED_ACCOUNT_ID = 0
 export const UNIFIED_INBOX_ID = 'unified'
 export const PRIORITY_INBOX_ID = 'priority'
 export const PAGE_SIZE = 20
-export const UNDO_DELAY = 7 * 1000
```

Comment:
you can leave the constant for better readability and assign the value of `TOAST_UNDO_TIMEOUT` here.

Reply:
but then it will not be used anywhere, you mean - export const UNDO_DELAY = TOAST_UNDO_TIMEOUT, and then use UNDO _DELAY, instead of using directly TOAST_UNDO_TIMEOUT on our actions?

Reply:
> then use UNDO _DELAY, instead of using directly TOAST_UNDO_TIMEOUT on our actions

yes.